src/EditorFeatures/VisualBasicTest/MakeTypeAbstract/MakeTypeAbstractTests.vb

```diff
@@ -135,5 +135,24 @@ Namespace NS
     End Class
 End Namespace")
         End Function
+
+        <WorkItem(54218, "https://github.com/dotnet/roslyn/issues/54218")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeTypeAbstract)>
+        Public Async Function TestMethod_PartialClass() As Task
+            Await TestInRegularAndScript1Async("
+Public Partial Class [|Foo|]
+    Public MustOverride Sub M()
+End Class
+
+Public Partial Class Foo
+End Class",
+"
+Partial Public MustInherit Class Foo
+    Public MustOverride Sub M()
+End Class
+
+Public Partial Class Foo
+End Class")
```

Comment:
Since the `Public` and `Partial` order was swapped here. Consider adding a test with demonstrating this respects `visual_basic_preferred_modifier_order` code-style option.

Reply:
Isn't the fact that the order was changed proof that the code-style option is being applied..
![thinking](https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)

Reply:
Looks like there is some deliberate ordering decisions going on in VBs syntax generator which I am definitely not qualified enough to argue with:
https://github.com/dotnet/roslyn/blob/main/src/Workspaces/VisualBasic/Portable/CodeGeneration/VisualBasicSyntaxGenerator.vb#L2503-L2504

Reply:
@CyrusNajmabadi Is SyntaxGenerator expected to respect code-style options?